UnleashClient/__init__.py

```diff
@@ -186,14 +196,44 @@ def is_enabled(self,
             except Exception as excep:
                 LOGGER.warning("Returning default value for feature: %s", feature_name)
                 LOGGER.warning("Error checking feature flag: %s", excep)
-
-                if fallback_function:
-                    fallback_value = default_value or fallback_function(feature_name, context)
-                else:
-                    fallback_value = default_value
-
-                return fallback_value
+                return self._get_fallback_value(default_value, fallback_function, feature_name, context)
         else:
             LOGGER.warning("Returning default value for feature: %s", feature_name)
             LOGGER.warning("Attempted to get feature_flag %s, but client wasn't initialized!", feature_name)
-            return default_value
+            return self._get_fallback_value(default_value, fallback_function, feature_name, context)
+
+
+    # pylint: disable=broad-except
+    def select_variant(self,
```

Comment:
other clients (Java, node, ruby) decided to call this method `getVariant`; Why did you choose `select_variant` over `get_variant`? To me select sounds like it might have a side-effect (like remembering the choice for a specific user)

Reply:
Sure, I can go with `get_varient()`!  I think I just badly misread part of the node code. :)

Reply:
Fixed!